db/app.js

```diff
@@ -1,9 +1,23 @@
 const express = require("express")
 const app = express()
-const { apiHealthCheck , getTopics } = require("./controller")
-
+app.use(express.json());
```

Comment:
Has this line of code been driven by a test? Every code written should be driven by a test or part of a test but we haven't yet got one that requires express.json() yet and therefore this code shouldn't exist at this point! :eyes: